bin/setup

```diff
@@ -12,12 +12,11 @@ echo -e "You may need to enter your root password...\n"
 sudo mkdir -p /etc/resolver
 echo -e "nameserver 127.0.0.1\nport 53" | sudo tee /etc/resolver/dev.gov.uk >/dev/null
 
-echo -e "✅ Configuring dnsmasq [/usr/local/etc/dnsmasq.conf]\n"
-echo "conf-dir=/usr/local/etc/dnsmasq.d,*.conf" > /usr/local/etc/dnsmasq.conf
+echo -e "✅ Configuring dnsmasq [$(brew --prefix)/etc/dnsmasq.conf]\n"
+echo "conf-dir=$(brew --prefix)/etc/dnsmasq.d,*.conf" >> $(brew --prefix)/etc/dnsmasq.conf
 
-echo -e "✅ Configuring dnsmasq [/usr/local/etc/dnsmasq.d/development.conf]\n"
-mkdir -p /usr/local/etc/dnsmasq.d
```

Comment:
Doesn't this directory still need to be created? Or was it already created by something else – and therefore unnecessary here?

The directory won't be auto-created if it doesn't already exist:

```bash
$ echo "hello" > missing-dir/file.txt
zsh: no such file or directory: missing-dir/file.txt
```

Reply:
Oh, that's a good point. It _doesn't_ need to be created in the /opt/homebrew/etc case (there's already one there). But I can put in the mkdir -p line safely for Intel systems, as you say.